mapper/mapper.go

```diff
@@ -709,6 +710,49 @@ func CreateTimePage(req *http.Request, f filter.Model, d dataset.DatasetDetails,
 		p.Data.SelectedEndMonth = selDates[len(selDates)-1].Month().String()
 		p.Data.SelectedEndYear = fmt.Sprintf("%d", selDates[len(selDates)-1].Year())
 	}
+	var minYear, maxYear string
+	var selectedMonths []string
+	for _, selVal := range selVals {
+		month, err := time.Parse("Jan-06", selVal.Option)
+		if err != nil {
+			log.Event(ctx, "unable to convert date to month value", log.ERROR, log.Error(err))
+			continue
```

Comment:
Same question here - what is the impact of this failing? Do we just have an empty value?